### ISSUE TITLE
could not find module for package import

### ISSUE THREAD

**Issue by 3052** [google/deps.dev#56]
this page:
https://deps.dev/go/154.pages.dev%2Fwidevine

shows this:

Warning
We found errors while resolving dependencies that may result in an incomplete or inaccurate dependency graph.
    could not find module for package import: 154.pages.dev/protobuf

which is not valid:

~~~
<!doctype html>
<html lang="en">
<head>
<meta name="go-import" content="154.pages.dev/protobuf git https://github.com/3052/protobuf">
</head>
</html>
~~~

https://pkg.go.dev/154.pages.dev/protobuf
https://godocs.io/154.pages.dev/protobuf

**Comment by adg** [comment]
Thanks for reporting this. Using this case we were able to diagnose and fix a bug in our data pipeline (related to custom module paths changing over time - the underlying repo used to be something else, right?).

**Comment by 3052** [comment]
yes, GitHub has this fun habit of banning my GitHub account without explanation, so securing my own domain and the `go-import` tag have been quite helpful in dealing with their crap.
